### DISCUSSION TITLE
Formula that installs Python modules _and_ scripts

### DISCUSSION THREAD

**Discussion by jackjansen** [Homebrew/discussions#3427] [Writing Formulae/Casks]
I have a package for which I'm writing a formula which consists of native libraries, native executables, Python packages and Python command line scripts.

All the native stuff (built with cmake) installs just fine into the Cellar. But I would like to install the Python packages into the "normal" python site-packages (because otherwise you can't import the packages from Python) and the scripts into my Cellar (because that seems to be where they belong).

I've also thought of installing everything into my Cellar (so including all the Python stuff), and then adding `.pth` to the Python installation so it'll pick up my packages.

I can't imagine I'm the first person who wants to do this, any suggestions for sample code to look at?

**Comment by carlocab** [comment]
What is the "normal" python site-packages?

Have you looked at formulae like `numpy`, `tbb`, `six`, or `protobuf`, where one can do (for example) `import numpy` with a Homebrew Python of the appropriate version?

**Reply by jackjansen** [reply]
Thanks, `numpy` is a great tip, it has some scripts as well. I've already learned from a quick inspection that they're running `setup.py` manually with prefix set to their cellar, now to find out how they do the magic to make the modules appear in Python.
